packages/api/src/providers/GroupActivities/private/createDefaultGroupActivitiesMiddleware.ts

```diff
@@ -19,7 +19,11 @@ function bin<T>(items: T[], grouping: (last: T, current: T) => boolean): T[][] {
     lastItem = item;
   }
 
-  return bins;
+  for (const bin in bins) {
+    Object.freeze(bin);
+  }
+
+  return Object.freeze(bins);
```

Comment:
Any benefit in feering here?

Reply:
Preventing someone from changing the order at any point of time in the future. 🤣